packages/ckeditor5-dev-webpack-plugin/lib/servetranslations.js

```diff
@@ -203,6 +203,12 @@ function getPathToPackage( cwd, resource, packageNamePattern ) {
 function getCompilationHooks( compiler, compilation ) {
 	const { webpack } = compiler;
 
+	// Webpack 4 may not expose its version.
+	if ( !webpack.version ) {
+		return compilation.hooks.normalModuleLoader;
+	}
```

Comment:
It looks like webpack 4 is not exposed in either `compiler` or `compilation` objects, so the `webpack` constant is `undefined` here. Then, checking the `webpack.version` throws an error. The whole logic could be simplified to just checking the existence of the `compiler.webpack` property, which is `undefined` in webpack 4, but it is present in webpack 5.